nelmio/api-doc-bundle/3.0/config/packages/nelmio_api_doc.yaml

```diff
@@ -0,0 +1,22 @@
+#nelmio_api_doc:
+#    exclude_sections:     []
+#    default_sections_opened:  true
+#    motd:
+#        template:             'NelmioApiDocBundle::Components/motd.html.twig'
+#    request_listener:
+#        enabled:              true
+#        parameter:            _doc
+#    swagger:
+#        api_base_path:        /api
+#        swagger_version:      '1.2'
+#        api_version:          '0.1'
+#        info:
+#            title:                Symfony2
+#            description:          'My awesome Symfony2 app!'
+#            TermsOfServiceUrl:    null
+#            contact:              null
+#            license:              null
+#            licenseUrl:           null
+#    cache:
+#        enabled:              false
+#        file:                 '%kernel.cache_dir%/api-doc.cache'
```

Comment:
Should be replaced by something like:
```yml
nelmio_api_doc:
#    documentation:
#        info:
#            title: My App
#            description: This is an awesome app!
#            version: 1.0.0
#    routes: # to filter documented routes
#        path_patterns: # an array of regexps
#            - ^/api

Reply:
IMO the default `/api/doc` route should be excluded from the swagger by defining the `path_patterns` in a format, something like `- ^/api/(?!doc)` because currently, it will display `/api/doc` route in the swagger which is un-needed.

Reply:
@takeit you're right, I updated my message